.gitlab-ci.yml

```diff
@@ -175,7 +175,7 @@ test:backend:integration:
   before_script:
     - apk add make bash git curl
     - *dind-login
-    - make -C backend -j 4 docker-pull
+    - make -C backend -j 4 docker-pull MENDER_IMAGE_TAG=$MENDER_IMAGE_TAG_TEST
```

Comment:
oh nice, it took me hours and you did it in 5s thank you; you are my hero.

Reply:
Sorry, I have on my todo list to write documentation for the new make(1) interface and how the new CI works, but I was trying to get everything in place first.